metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -187,6 +187,12 @@ def argo_workflows(obj, name=None):
     default=None,
     help="Incident.io Alert source config ID. Example '01GW2G3V0S59R238FAHPDS1R66'",
 )
+@click.option(
+    "--incident-io-metadata",
+    default=None,
+    type=JSONTypeClass(),
+    help="Incident.io Alert Custom Metadata fields.",
+)
```

Comment:
rather than json - can this be `--incident-io-metadata key=value --incident-io-metadata key2=value2 ...`

Reply:
definitely can be changed to that for nicer ux. I was also wondering whether the value should be settable through the Metaflow profile, or is the CLI option enough for now?

Reply:
CLI option is enough for now